agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java

```diff
@@ -812,8 +812,8 @@ private void exportEvents(
         setOperationName(telemetryBuilder, span.getAttributes());
       }
       setTime(telemetryBuilder, event.getEpochNanos());
-      setExtraAttributes(telemetryBuilder, event.getAttributes());
       setSampleRate(telemetryBuilder, samplingPercentage);
+      setExtraAttributes(telemetryBuilder, event.getAttributes());
```

Comment:
does the order matter?

Reply:
no, I just re-ordered this one for consistency with the rest